eng/pipelines/templates/pipeline.yml

```diff
@@ -1,107 +1,3 @@
-
-parameters:
-- name: isOfficialBuild
-  type: boolean
-  default: false
-- name: DartLabEnvironment
-  displayName: DartLab Environment
-  type: string
-  default: Production
-  values:
-  - Production
-  - Staging
-- name: E2EPart1AgentCleanup
-  displayName: Delete or keep E2E Part 1 machine for debugging
-  type: string
-  default: delete
-  values:
-  - delete
-  - stop
-- name: E2EPart2AgentCleanup
-  displayName: Delete or keep E2E Part 2 machine for debugging
-  type: string
-  default: delete
-  values:
-  - delete
-  - stop
-- name: ApexAgentCleanup
-  displayName: Delete or keep VS Apex test machine for debugging
-  type: string
-  default: delete
-  values:
-  - delete
-  - stop
-- name: NuGetLocalizationType
-  displayName: Whether to do production-ready localization (Full), or pseudo-localization, aka PLOC, (Pseudo) for testing.
-  type: string
-  default: Full
-  values:
-  - Full
-  - Pseudo
-- name: RunApexTests
-  displayName: Run Apex tests
-  type: boolean
-  default: true
-- name: RunBuildForPublishing
-  displayName: Build bits for publishing
-  type: boolean
-  default: true
-- name: RunCrossFrameworkTestsOnWindows
-  displayName: Run cross framweork tests on Windows
-  type: boolean
-  default: true
-- name: RunEndToEndTests
-  displayName: Run end-to-end tests
-  type: boolean
-  default: true
-- name: RunFunctionalTestsOnWindows
-  displayName: Run functional tests on Windows
-  type: boolean
-  default: true
-- name: RunSourceBuild
-  displayName: Run source build
-  type: boolean
-  default: true
-- name: RunTestsOnLinux
-  displayName: Run tests on Linux
-  type: boolean
-  default: true
-- name: RunTestsOnMac
-  displayName: Run tests on Mac
-  type: boolean
-  default: true
-- name: RunMonoTestsOnMac
-  displayName: Run Mono tests on Mac
-  type: boolean
-  default: false
-
-resources:
-  pipelines:
-  - pipeline: DartLab
-    source: DartLab
-    branch: main
-  repositories:
-  - repository: DartLabTemplates
-    type: git
-    name: DartLab.Templates
-    ref: refs/heads/main
-
-variables:
```

Comment:
I don't know if variables need to be set in the "entry point" or "top level" yaml. If it does, then these variables need to be copied into `official.yaml` as well.

However, try reverting all the changes to this file, and removing the resources and variables from `pull_request.yaml`.

I believe that this file shouldn't need to be modified at all for this PR.

Reply:
Answered in https://github.com/NuGet/NuGet.Client/pull/5632/files#r1498420606